src/Tools/AnalyzerRunner/Program.cs

```diff
@@ -106,7 +106,7 @@ public static async Task Main(string[] args)
             {
                 if (!string.IsNullOrEmpty(options.ProfileRoot))
                 {
-                    ProfileOptimization.StartProfile(nameof(Microsoft.CodeAnalysis.SolutionCrawler.IIncrementalAnalyzer));
+                    ProfileOptimization.StartProfile("IncrementalAnalyzer");
```

Comment:
Is this still required? Can't we delete the incremental analyzer runner completely? Can be done in a follow-up PR though..

Reply:
yeah.  can delete later.